package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "morgan": "^1.9.1",
+    "open": "0.0.5",
```

Comment:
Oof. Can you `npm add compression express` too? I didn't add these packages after I wiped the first attempt at this project and re-added.

Reply:
Done 👍